UKHO.FmEssFssMock.API/UKHO.FmEssFssMock.API/Services/FileShareService.cs

```diff
@@ -107,7 +107,7 @@ public BatchDetail GetBatchDetails(string batchId, string homeDirectoryPath)
             return new BatchDetail
             {
                 BatchId = batchId,
-                Status = "Committed",
+                Status = File.Exists(Path.Combine(path, "CommitInProgress.txt")) ? "CommitInProgress" : "Committed",
```

Comment:
Please create a helper function for this:

        private string GetBatchStatus(string path) => File.Exists(Path.Combine(path, "CommitInProgress.txt")) ? "CommitInProgress" : "Committed";

It can be called further down in this class as well to avoid duplicate code.

Reply:
Added private method and replaced